patsicko/my-brand-bakend
# Ft blog update 008

## What this PR do
Update an existing blog
## Description of the tasks to be completed
- PUT /api/updateBlog
[Finishes#008]

## How should this be manually tested
- open postman
- set your HTTP request to PUT
- input API endpoint
- switch to the Body tab
- Click raw
- select JSON
-  create a json object with blogTitle, blogAutor,blogImane and blogText for updated blog
- Click Send.
- Status: 200 ok should be displayed
- Updated data are showing up in the body.


## Trello story id
#008
